packages/manager/apps/pci-savings-plan/src/components/Commitment/Commitment.tsx

```diff
@@ -47,7 +47,11 @@ const Commitment = ({
       rounded
       inline
       variant={ODS_TILE_VARIANT.stroked}
-      className="flex flex-row items-center mr-5 my-4 justify-between w-full cursor-pointer"
+      className={`flex flex-row items-center mr-5 my-4 justify-between w-full cursor-pointer ${
```

Comment:
As a suggestion, it will be better to use `clsx` to build conditional classNames, in particular to avoid interpolation and the ternary which returns an empty string in the className if the condition is not met